GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -2150,9 +2140,7 @@ private void ViewFileHistoryMenuItem_Click(object sender, EventArgs e)
                 return;
 
             if (list.SelectedItems.Count() == 1)
-            {
```

Comment:
It would be easy to implement this as a follow-up change. I have a set of analyzers which can enforce several different policies related to the preferred code style for braces, and then fix the entire code base to match those settings.